config.js

```diff
@@ -22,7 +23,7 @@ module.exports = {
 		client: "client",
 		render: "render",
 		templates: "client/hbs",
-		upload: "upload"
+		upload: "upload",
```

Comment:
-1 for trailing commas.  I understand the argument behind them but don't find it compelling.